fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -2607,4 +2622,13 @@ private List<Expr> getDistributeExpr(List<ExprId> childOutputIds, DistributionSp
         }
         return Lists.newArrayList();
     }
+
+    private void checkInnerGroupCommit(List<ScanNode> scanNodes) {
```

Comment:
add comment to explain the diff between GroupCommitTableValuedFunction and other tvf